ToggleMachines/mod.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Id": "ToggleMachines",
+  "DisplayName": "Toggle Machines",
+  "Author": "AHK1221",
+  "Version": "1.0.4",
+  "Enable": true,
+  "AssemblyName": "ToggleMachines.dll",
+  "EntryMethod": "ToggleMachines.Qpatch.Patch",

```

Comment:
If you are using the `[QModCore]` and `[QModPatch]` attributes from  QModManager 3.0,  
then you don't need to include the `EntryMethod` line in the mod.json.  
QModManager will use these attributes to find the patch method.

I would also suggest removing the `Config` entry if it isn't needed.

You would end up with something like this:
```JS
{
  "Id": "ToggleMachines",
  "DisplayName": "Toggle Machines",
  "Author": "AHK1221",
  "Version": "1.0.4",
  "Enable": true,
  "AssemblyName": "ToggleMachines.dll"
}
```
Remember to remove any trailing comma `,` on the final entry before the closing curly bracket `}`